### PR TITLE
To Do list: list structure

### PR DESCRIPTION
**Pull request by HENRYKC24**
### In This Pull Request, I:
- Set up a new project with webpack
- Created an index.html file
- Created an empty To Do List placeholder
- Created an index.js file and set an array of some simple to do tasks 
- Wrote a function to iterate over the tasks array and populate an HTML list item element for each task
- Rendered the dynamically created list of tasks in the dedicated placeholder on page load
- Created a style.css and set rules for the To Do List

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+      toBeDeleted.push(index);
+    } else {
+      nextToCheckBox.style.textDecorationLine = 'none';
+      nextToCheckBox.style.color = 'rgb(150, 145, 145)';
+      toBeDeleted.splice(toBeDeleted.indexOf(index), 1);
+    }
+  });
+  div2.appendChild(input);
+
+  const input2 = document.createElement('input');
```
Comment (rloterh) [inline src/index.js:57]: - [ ] Your _js_ module does not include `array object` of `tasks`. Kindly ensure that it has an array of some simple to do tasks (array of objects) with each object containing three keys - `description [string]`, `completed [bool]`, `index: [number]`. 
- [ ] Also, please ensure that on _page load_, the task array is rendered **dynamically** to the DOM. The tasks should appear in order of the `index` values for each task

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Henry, 

You did extremely great on this project!👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there!👍 

## Good Points 👍  
- Correct project structure
- No linter errors


## Required Changes ♻️
- _Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

-  _Kindly consider my comments marked **[OPTIONAL]**:_
- [ ] OPTIONAL 
Your _linters_ is raising an error, kindly endeavor to fix this issue to make your repository appear professional

Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (HENRYKC24) [reply src/index.js:57]: That has been done. Thanks @rloterh:pray:

**Review by mimipeshy — APPROVED** [review]
## Hi @HENRYKC24  👋🏾 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](<https://microverse.zendesk.com/hc/en-us/articles/1500004088561>)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](<https://airtable.com/shrQAqnBwek5a0O0s>)._

**Comment by HENRYKC24** [comment]
Thank you so much, @mimipeshy :pray:

**Pull request merged**